Header header

float64 depth
float64 offset
float64 range